test/asyncio_tests/test_examples.py

```diff
@@ -1107,3 +1108,31 @@ async def test_causal_consistency(self):
             async for item in items.find({'end': None}, session=s2):
                 print(item)
         # End Causal Consistency Example 2
+
+    @env.require_version_min(4, 7)
+    @asyncio_test
+    def test_versioned_api(self):
+        # Versioned API examples
+        AsyncIOMotorClient = lambda _, server_api: self.asyncio_client(
+            server_api=server_api, connect=False)
```

Comment:
I wrote this to mimic the PyMongo PR - https://github.com/mongodb/mongo-python-driver/commit/048f54ddde999459d27ff6d5a6fc5db0de28a490 
But I have no idea why we set `connect=False`? Can we remove it?

Reply:
It's to reduce the overhead of the test. Since the test doesn't need to actually use the MongoClient there's no point in starting all the monitor threads. Let's keep it and add a comment.

Reply:
Ok